### ISSUE TITLE
Compatability mode does not work on all motherboards

### ISSUE THREAD

**Issue by SenlisO** [drauger-os-development/systemd-boot-manager#3]
Compatibility mode should make it where every motherboard can boot DraugerOS via UEFI, but it doesn't always work.

**Comment by SenlisO** [comment]
Have we had anyone else report this issue since I created it a year ago?

**Comment by Batcastle** [comment]
Yes. It's less common, but it does still happen. We need someone on the dev team with a device that persistently has this issue.
